### PR TITLE
fix [TOML Configus] TenantDomainDropDownEnabled in application-authen…

### PR DESCRIPTION
**Pull request by wijith7**
fix - https://github.com/wso2/product-is/issues/6133

### CODE REVIEW

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2 [hunk 1]
```diff
@@ -151,4 +151,16 @@
         <AuthenticationEndpointRedirectParam name="{{parameter}}"/>
         {% endfor %}
     </AuthenticationEndpointRedirectParams>
+    {% if tenant.domainDropDown.enable is defined %}
```
Comment (hasinidilanka) [inline features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2]: Change the key domainDropDown to domain_drop_down

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2 [hunk 2]
```diff
@@ -151,4 +151,16 @@
...
+      <TenantDomainDropDownEnabled>{{tenant.domainDropDown.enable}}</TenantDomainDropDownEnabled>
+    {% endif %}
+    {% if oauth.custom_grant_type is defined %}
```
Comment (hasinidilanka) [inline features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2]: This should be corrected as below:
 {% if tenant.data_listener_urls is defined %}
      <TenantDataListenerURLs>
      {% for listener in tenant.data_listener_urls %}   
           <TenantDataListenerURL>
           {{listener.url}}
           </TenantDataListenerURL>     
       {% endfor %}
        </TenantDataListenerURLs>
     {% endif %}

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2 [hunk 3]
```diff
@@ -151,16 +151,16 @@
...
+      <TenantDomainDropDownEnabled>{{tenant.domain_drop_down.enable}}</TenantDomainDropDownEnabled>
     {% endif %}
-    {% if oauth.custom_grant_type is defined %}
-      {% for listener in oauth.custom_grant_type %}
-         <TenantDataListenerURLs>
+    {% if tenant.data_listener_urls is defined %}
+       <TenantDataListenerURLs>
+         {% for listener in tenant.data_listener_urls %}
            <TenantDataListenerURL>
            {{listener.url}}
```
Comment (hasinidilanka) [inline features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2]: This should be {{listener}}

Comment (wijith7) [reply features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2]: done

**Pull request merged**